src/screensComponents/modal/index.js

```diff
@@ -58,17 +59,22 @@ export default class Modal extends BaseComponent {
   }
 
   render() {
-    const {blurView, enableModalBlur, visible, ...others} = this.props;
+    const {blurView, enableModalBlur, visible, overlayBackgroundColor, ...others} = this.props;
     const defaultContainer = enableModalBlur && Constants.isIOS ? BlurView : View;
     const Container = blurView ? blurView : defaultContainer;
 
     return (
-      <RNModal visible={Boolean(visible)} {...others}>
-        <Container style={{flex: 1}} blurType="light">
-          {this.renderTouchableOverlay()}
-          {this.props.children}
-        </Container>
-      </RNModal>
+      <View>
+        {Constants.isAndroid && Boolean(visible) && !!overlayBackgroundColor && (
```

Comment:
There's an issue, after dismissing the modal the status bar color stays

Reply:
It is a bit weird, it works fine for me in regular condition, but if I kill the application and opens it again it has this bug. Do you have an idea?

Reply:
It seems that the <StatusBar/> component is buggy:
https://github.com/react-native-community/react-native-statusbar/issues/20
I'll upload a fix.

Reply:
Two notes to consider:
1. I've changed the Modal to extend Component and not BaseComponent (I am using a new lifecycle method).
2. I'm using ThemeManager.primatyColor, it looks ok on both apps, but it might need further testing.